tidb-binlog/pump_client/bench_test.go

```diff
@@ -57,21 +58,15 @@ func writeFakeBinlog(b *testing.B, pumpClient *PumpsClient, threadCount int, num
 					StartTs: int64(baseTS + j),
 				}
 				err := pumpClient.WriteBinlog(pBinlog)
-				if err != nil {
-					b.Error(err)
-					return
-				}
+				assert.Nil(b, err)
```

Comment:
```suggestion
				require.NoError(b, err)
```

There is a `NoError` assertion to use. Ditto others.

Reply:
done